src/types/rpc.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Type definitions for RPC methods.
```

Comment:
I suggest to clarify that this refers to the frame that embeds the host frame - perhaps give the LMS app as a concrete use case, since we also have RPC methods for communication between the host, sidebar and guest frames in the client.